io/eolearn/io/sentinelhub_service.py

```diff
@@ -117,8 +116,7 @@ def _prepare_request_data(self, eopatch, bbox, time_interval):
         return {
             'layer': self.layer,
             'bbox': bbox if bbox is not None else self._get_parameter('bbox', eopatch),
-            'time': datetime_to_iso(time_interval) if isinstance(time_interval, str) else [
-                datetime_to_iso(x) if not isinstance(x, str) else x for x in time_interval],
+            'time': time_interval,
```

Comment:
Can it happen that the time interval in the existing EOPatch is of the format (`str`, `str`)? Previously if the time interval was (`str`, `str`) it would get transformed to (`datetime`, `datetime`). If it already was the latter format, then it was just copied.

Reply:
time was before always (str, str), now it can be both